third-party/realdds/include/realdds/topics/dds-topic-names.h

```diff
@@ -21,5 +24,179 @@ constexpr char const * NOTIFICATION_TOPIC_NAME = "/notification";
 constexpr char const * CONTROL_TOPIC_NAME = "/control";
 constexpr char const * METADATA_TOPIC_NAME = "/metadata";
 
+
+namespace notification {
+    namespace key {
```

Comment:
`key` have many meanings. I guess you refer to a json dictionary key, but maybe `field` is better suited here? e.g. `stream_header::field::name`

Reply:
Yes, key-value pairs. In json it's called Key as well.
You like field better?

Reply:
You can leave key if you prefer